index.html

```diff
@@ -134,6 +134,26 @@
             </div>
         </div>
         <hr>
+                <div class="featurette white">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-9">
+                        <h1>Peer reviewed codes using <span class="alpscore"></span></h1>
+                        <p>
+Several applications use <span class="alpscore"></span> as their underlying library. Here is a selection of codes and publications 
+                        </p>
+                    </div>
+                    <div class="col-md-3">
+                        <p><a class="btn btn-primary" href="https://github.com/ALPSCore/CT-HYB-SEGMENT" role="button"><i class="fa fa-book fa-fw"></i>&nbsp; Hybridization Expansion (Segment)</a></p>
```

Comment:
The links look like buttons on the right side of the page. Is it intentional?

Reply:
Intentional: yes, but also ugly. Can you propose something better?

Reply:
Thinking on it. 
Maybe, a single button "Codes that use ALPSCore" whith a link to the Wiki page?

Reply:
For now, in the section "Using ALPSCore", replace the "Mailing list" (it is not used by anyone anyway) with "Applications that use ALPSCore" button.

Later, add another section to the page: "Outreach" or "For ALPSCore Users" something like that.
Move the following buttons there:
1) Applications that use ALPSCore
2) Tutorials
3) Screencasts (TBD)

Reply:
OK, that makes sense. can you commit the changes or should I?